packages/flutter_tools/test/general.shard/build_system/targets/ios_test.dart

```diff
@@ -696,12 +678,19 @@ void main() {
         defines: <String, String>{
           kIosArchs: 'arm64',
           kSdkRoot: 'path/to/iPhoneOS.sdk',
-          kBitcodeFlag: '',
         },
       );
 
       processManager.addCommands(<FakeCommand>[
-        copyPhysicalFrameworkCommand,
+        FakeCommand(command: <String>[
+          'rsync',
+          '-av',
+          '--delete',
+          '--filter',
+          '- .DS_Store/',
+          'Artifact.flutterFramework.TargetPlatform.ios.release.EnvironmentType.physical',
```

Comment:
`fails when bitcode strip fails` test will only pass during `ReleaseUnpackIOS` since stripping will now only happen on release, not dependent on `kBitcodeFlag`.